Examples/ObjectiveC/ShapeCollectionFeatureExample.m

```diff
@@ -0,0 +1,56 @@
+#import "ShapeCollectionFeatureExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleShapeCollectionFeature = @"ShapeCollectionFeatureExample";
+
+@interface ShapeCollectionFeatureExample () <MGLMapViewDelegate>
+
+@end
+
+@implementation ShapeCollectionFeatureExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    MGLMapView *mapView = [[MGLMapView alloc] initWithFrame: self.view.bounds styleURL: [MGLStyle lightStyleURLWithVersion:9]];
+    mapView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
+    
+    [mapView setCenterCoordinate:CLLocationCoordinate2DMake(38.897435, -77.039679) zoomLevel:12 animated:NO];
+    mapView.delegate = self;
+    
+    [self.view addSubview:mapView];
+}
+
+- (void)mapView:(MGLMapView *)mapView didFinishLoadingStyle:(MGLStyle *)style {
+    
+    NSURL *url = [[NSURL alloc] initWithString: @"https://api.mapbox.com/datasets/v1/mapbox/cj004g2ay04vj2xls3oqdu2ou/features?access_token=pk.eyJ1IjoibWFwYm94IiwiYSI6ImNpemc0YWlpNzAwcXUyd21ldDV6OWpxMGwifQ.A92RQZpwUgtGtCmdSE4-ow"];
+    if (url) {
+        NSData *data = [[NSData alloc] initWithContentsOfURL:url];
+        MGLShape *feature = [MGLShape shapeWithData:data encoding: NSUTF8StringEncoding error: nil];
+        
+        // Create source and add it to the map style.
+        MGLShapeSource *source = [[MGLShapeSource alloc] initWithIdentifier:@"transit" shape:feature options:nil];
+        [style addSource:source];
+        
+        // Create station style layer.
+        MGLCircleStyleLayer *circleLayer = [[MGLCircleStyleLayer alloc] initWithIdentifier:@"stations" source:source];
+        circleLayer.predicate = [NSPredicate predicateWithFormat:@"%@" argumentArray:@[@"Station"]];
```

Comment:
This isn’t a valid predicate. A valid predicate must include an operator. Based on the data in the dataset, I suspect you want to check whether the `TYPE` attribute is `Station`. Here are some ways of expressing that:

```objc
[NSPredicate predicateWithFormat:@"TYPE = 'Station'"]
[NSPredicate predicateWithFormat:@"TYPE == 'Station'"]
[NSPredicate predicateWithFormat:@"TYPE = %@", @"Station"]
[NSPredicate predicateWithFormat:@"TYPE IN {'Station'}"]
[NSPredicate predicateWithFormat:@"TYPE IN %@", @[@"Station"]]
[NSPredicate predicateWithFormat:@"{'Station'} CONTAINS TYPE"]
```

Note that I’m using `+[NSPredicate predicateWithFormat:]` here instead of `+[NSPredicate predicateWithFormat:argumentArray:]`. The latter is useful if you’re given an array to put in the second parameter and would find it inconvenient to have to lump it in with the format in the first parameter.